packages/backend/src/database/migrations/20240108152115_group_attributes.ts

```diff
@@ -0,0 +1,30 @@
+import { Knex } from 'knex';
+
+const userAttributesTableName = 'user_attributes';
+const groupUserAttributes = 'group_user_attributes';
+
+export async function up(knex: Knex): Promise<void> {
+    if (!(await knex.schema.hasTable(groupUserAttributes))) {
+        await knex.schema.createTable(groupUserAttributes, (table) => {
+            table.uuid('group_uuid').notNullable();
+            table.integer('organization_id').notNullable();
```

Comment:
I think we need this to be a foreign key and delete on cascade. 

Example code from : packages/backend/src/database/migrations/20230714124708_user_attributes.ts

```
table
                .integer('organization_id')
                .notNullable()
                .references('organization_id')
                .inTable('organizations')
                .onDelete('CASCADE');
```

Reply:
I just changed this a little after looking around and learning a bit more. What do you think of how it is now? No more org id. Just the mapping between group and attribute + a value

Reply:
I think that makes sense since the `user_attributes` table already has that relation.